be/src/storage/tablet_meta.h

```diff
@@ -166,6 +172,11 @@ class TabletMeta {
 
     Status set_partition_id(int64_t partition_id);
 
+    void set_enable_binlog(bool enable_binlog);
```

Comment:
Why need these methods considering it already provides `set_binlog_config` methods?

Reply:
The previous consideration was to only modify a single attribute of binlog, and FE would only send the relevant modified parameter to BE, so these methods were added. At present, the logic is unified, any parameter is modified, and the entire Config is sent. I will delete them